src/CONST.ts

```diff
@@ -1363,6 +1363,9 @@ const CONST = {
         MAKE_REQUEST_WITH_SIDE_EFFECTS: 'makeRequestWithSideEffects',
     },
 
+    MAP_PADDING: 50,
```

Comment:
Looks like there is some duplicate effort being done to this in https://github.com/Expensify/App/pull/25977/files#diff-fccfa43127fe02d958452f91303d6a59a20e60c031c0d7d920ac6175a37c83c0R2614-R2619

Let's be sure to coordinate those changes so we don't end up with two different constants.

Reply:
I'll keep track of this and make sure there is only one constant for each purpose